nixos/modules/flyingcircus/roles/redis.nix

```diff
@@ -4,6 +4,14 @@ let
   cfg = config.flyingcircus.roles.redis;
   fclib = import ../lib;
 
+  # This looks clunky.
+  redis_package =
+    if config.flyingcircus.roles.redis.enable
+    then pkgs.redis
+    else if config.flyingcircus.roles.redis4.enable
+    then pkgs.redis4
+    else null;
+
```

Comment:
Don't like it. I'd rather go with fc.roles.redis.enable and fc.roles.redis.package. Let's discuss this f2f.